packages/server/lib/gui/files.ts

```diff
@@ -2,40 +2,30 @@ import openProject from '../open_project'
 import { createFile } from '../util/spec_writer'
 import { showSaveDialog } from './dialog'
 
-export const showDialogAndCreateSpec = () => {
```

Comment:
`getConfig` no longer returns a promise..., so no more `then`, and no more `tap`, so I just made it into a regular old function, replacing `then` with `await`.